style.css

```diff
@@ -113,14 +210,16 @@ nav ul {
     right: 1em;
 }
 
-.text-with-icon svg {
-    position: absolute;
-    width: 1em;
-    fill: var(--gray-2);
+.icon-top {
+    top: 0;
+}
+
+.icon-center svg {
```

Comment:
Let's use BEM naming from the class and rename to `text-with-icon--center`. This way we know we have a "center" modifier for the `text-with-icon`.